tox.ini

```diff
@@ -75,3 +69,16 @@ commands_pre =
 commands =
     isort -rc .
     black .
+
+[testenv:spark]
+description = run the test driver with {basepython}
+passenv = JAVA_HOME
+commands_pre = python -m spacy download en_core_web_sm
+commands = python -m pytest -m spark {posargs}
+
+[testenv:complex]
+description = run the test driver with {basepython}
+passenv = JAVA_HOME
+commands_pre = python -m spacy download en_core_web_sm
+commands = python -m pytest -m complex {posargs}
```

Comment:
Actually can we do `complex and not spark` here so that we don't rerun. This makes Spark special here, but it is because it requires Java.